Dockerfile

```diff
@@ -0,0 +1,16 @@
+ARG GO_VERSION=1.17
+
+FROM golang:${GO_VERSION}-alpine as build
+
+RUN apk --no-cache add make ca-certificates 
+WORKDIR /src
+COPY go.mod go.sum /src/
+RUN go mod download
+COPY . /src/
+RUN make build
+ENTRYPOINT [ "/src/bin/in-toto" ]
```

Comment:
I think you can remove  the ENTRYPOINT here.. it's just a build layer.

Reply:
Removed EntryPoint as its not needed.